### PR TITLE
Calendar created

### PR DESCRIPTION
**Pull request by bn-maksymmaliuk**
[DEMO LINK](https://MaksymMaliuk.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: BEM doesn't allow us to do so. Better add styling to the parent element.

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  &__day {
+    @include square;
+    border: 1px solid black;
+    line-height: $cell-size;
+    text-align: center;
+    background-color: #eee;
+
+    transition: transform 0.3s, background-color 0.3s;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: Wrong duration - needs to be half of a second.
You can create a variable for it.

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    &:hover {
+      cursor: pointer;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: Set this prop to `&__day` element

File: src/styles/utils/_functions.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+@function calendarSize($cell-size, $padding, $gap) {
+  @return $cell-size * 7 + $gap * 6 + $padding * 2;
```
Comment (sergii-nosachenko) [inline src/styles/utils/_functions.scss]: Accept 7 as an argument. 6 can be calculated from it.

File: src/styles/utils/_reset.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```
Comment (sergii-nosachenko) [inline src/styles/utils/_reset.scss]: These rules belong to normalize

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**